### DISCUSSION TITLE
.net support

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [tursodatabase/libsql#442] [Q&A] [answered]
The libSQL API links in the getting started section of the readme.md file are broken.

Are there any plans for the .net and if no can i help?

**Comment by notrab** [comment, marked as answer]
Hey @fredericoregateiro, apologies for the delay. There's an awesome project with libsql and dotenv happening in [this repo](https://github.com/tvandinther/libsql-client-dotnet). If you join us on Discord, there's a dedicated channel you can use to dive in and help 😍

**Reply by fredericoregateiro** [reply]
Thanks @notrab i will check those
